OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -103,6 +105,9 @@ public Bullet(BulletInfo info, ProjectileArgs args)
 			}
 
 			smokeTicks = info.TrailDelay;
+
+			if (!info.High && speed.Range > 1024)
```

Comment:
If the bullet is travelling diagonally across the corner of a cell then it may pass across a wall with a speed < 1024.